src/components/Editor/Runners/PythonRunner/PyodideRunner/PyodideRunner.jsx

```diff
@@ -120,12 +120,15 @@ const PyodideRunner = (props) => {
   }, [codeRunStopped]);
 
   const handleLoading = () => {
-    dispatch(loadingRunner());
+    return;
   };
 
   const handleLoaded = (stdin, interrupt) => {
     stdinBuffer.current = stdin;
     interruptBuffer.current = interrupt;
+    if (loadedRunner !== "pyodide") {
+      dispatch(setLoadedRunner("pyodide"));
```

Comment:
I'm sure there's a good reason but this does look a little strange, are we happy there are no repercussions for doing this?

Reply:
I'll try and refactor in a separate ticket but I think this select/loaded runners approach works nicely and ensures its only loaded once and allows us to enable things like the run button when a runner is loaded. I need to give it some thought to tidy it up but it could definitely do with some enums at the very least